core/shared/src/main/scala/org/http4s/internal/CharPredicate.scala

```diff
@@ -347,3 +325,26 @@ object CharPredicate {
 
   // scalafix:on
 }
+
+class ApplyMagnet(val predicate: CharPredicate)
```

Comment:
This whole machinery was too cute anyway, in my opinion.  But this is fine... it's just private, and not used that widely.